principia.props

```diff
@@ -98,14 +99,16 @@
         -Xclang --system-header-prefix=gtest/
         -Xclang --system-header-prefix=gmock/
       </AdditionalOptions>
+      <MultiProcessorCompilation>true</MultiProcessorCompilation>
       <PreprocessorDefinitions Condition="!$(PrincipiaCompilerClangLLVM)">
         _ENABLE_EXTENDED_ALIGNED_STORAGE;NOGDI;NOMINMAX;%(PreprocessorDefinitions)
       </PreprocessorDefinitions>
       <TreatWarningAsError Condition="!$(PrincipiaCompilerClangLLVM)">true</TreatWarningAsError>
       <SDLCheck>true</SDLCheck>
     </ClCompile>
     <Link>
-      <GenerateDebugInformation>true</GenerateDebugInformation>
+      <GenerateDebugInformation>DebugFastLink</GenerateDebugInformation>
+      <LinkTimeCodeGeneration>UseFastLinkTimeCodeGeneration</LinkTimeCodeGeneration>
```

Comment:
Where is that from? I don't see it in https://blogs.msdn.microsoft.com/vcblog/2016/10/26/recommendations-to-speed-c-builds-in-visual-studio/.

Reply:
Changed to "Default".  Also moved incremental linking at a place where it actually has an effect.